models/models.comments.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../db/connection');
+
+exports.deleteCommentById = (comment_id) => {
```

Comment:
You should only use the request method (eg "delete") for the controller name. Use a similar verb eg "remove" for models.